### PR TITLE
Properly delete contact groups

### PR DESCRIPTION
**Pull request by y-lohse**
The groups where not deleted in the right field, which was causing problems.

There's still the underlying question of why groups seem to get saved in the `relationships` AND `groups` property, which I haven't investigated yet.

### CODE REVIEW

File: src/thunks/cleanTrashedGroups.js [hunk 1]
```diff
@@ -18,10 +18,14 @@ const cleanTrashedGroups = () => async (dispatch, getState, { client }) => {
...
-          _id: groupId,
-          _type: DOCTYPE_CONTACT_GROUPS
+      relationships: {
+        groups: {
+          data: {
+            $elemMatch: {
+              _id: groupId,
+              _type: DOCTYPE_CONTACT_GROUPS
+            }
+          }
```
Comment (Crash--) [inline src/thunks/cleanTrashedGroups.js]: I think this is where and why you need the use of `.includes('groups')`, nop ?

Comment (y-lohse) [reply src/thunks/cleanTrashedGroups.js]: No, I don't actually care about the groups themselves right there, I just need the ids. Either that or I misunderstood what ypu meant.

**Comment by y-lohse** [comment]
@Crash-- I updated the PR — we're not manipulating the `relationships` object anymore, but we still have to query it manually.
I made the changes so we don't need to update cozy-client, but I'll create issues there to see what is best long term.

**Comment by y-lohse** [comment]
[(3) Groupes - supprimer](https://trello.com/c/AUW3POQY/1404-3-groupes-supprimer)

**Pull request merged**